react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt

```diff
@@ -61,4 +63,18 @@ internal class PaywallFooterViewManager : SimpleViewManager<PaywallFooterView>()
         return paywallFooterView
     }
 
+    @ReactProp(name = "options")
+    fun setOptions(view: PaywallFooterView, options: ReadableMap?) {
+        options?.let { props ->
+            if (props.hasKey("offering")) {
+                props.getDynamic("offering").asMap()?.let { offeringMap ->
+                    if (offeringMap.hasKey("identifier")) {
+                        view.setOfferingId(offeringMap.getString("identifier"))
```

Comment:
I am wondering, once we add other properties, will this cause the UI to reload multiple times? I was originally imagining to have multiple setters, then a method to "reload"... But maybe this is ok, if it becomes a performance issue we can iterate later.

Reply:
Yes, I had the `reload` function then removed it because we don't need it yet since we only have one parameter right now and felt I was over-engineering it and it required an update in purchases-android